src/dbcp/transform/local_opposition.py

```diff
@@ -96,6 +96,9 @@ def _transform_local_ordinances(local_ord_df: pd.DataFrame) -> pd.DataFrame:
     # manual corrections
     location_corrections = {
         "Batavia Township (Clermont County)": "Batavia Township (Branch County)",
+        "Town of Albion (Kennebec County)": "Albion (Kennebec County)",
+        "Town of Lovell (Oxford County)": "Lovell (Oxford County)",
+        "Town of Charlton (Worcester County)": "Charlton (Worcester County)",
```

Comment:
Got a null violation constraint in the CI because it had a fresh cache. I'm not sure why my older local cache was able to geocode these, but making these corrections worked.